src/backend/distributed/utils/namespace_utils.c

```diff
@@ -0,0 +1,35 @@
+/*-------------------------------------------------------------------------
+ *
+ * namespace_utils.c
+ *
+ * Utility functions related to namespace changes.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "catalog/namespace.h"
+#include "distributed/namespace_utils.h"
+
+/*
+ * PushAndNullifySearchPath pushes search_path to be NIL and sets addCatalog to
+ * true so that all objects outside of pg_catalog will be schema-prefixed.
+ * Afterwards, PopOverrideSearchPath can be used to revert the search_path back.
+ */
+void
+PushAndNullifySearchPath()
```

Comment:
Maybe NullifyAndPushSearchPath since we push after we set it to nil

Reply:
Suggestion: `PushOverrideEmptySearchPath(MemoryContext)` to be more compatible with the original function